openshift/templates/bc-wallet-demo-web/bc-wallet-demo-web-deploy.yaml

```diff
@@ -1,16 +1,18 @@
----
+apiVersion: v1
```

Comment:
You should have been getting warnings about this.  I believe it has to be `template.openshift.io/v1`